docs/tutorial/tut_multi_active_satellites.md

```diff
@@ -0,0 +1,176 @@
+    ```
+
+To create a MAS model, we simply copy and paste the above template into a model named after the MAS we
+are creating. dbtvault will generate a MAS using parameters provided in the next steps.
+
+MAS should use the incremental materialization, as we load and add new records to the existing data set. 
+
+We recommend setting the `incremental` materialization on all of your MAS using the `dbt_project.yml` file:
+
+=== "dbt_project.yml"
+
+    ```yaml
+    models:
+      my_dbtvault_project:
+       multi_active_satellites:
+        materialized: incremental # See tip below
+        tags:
+          - ma_sat
+        ma_sat_customer_details:
+          vars:
+            ...
+            ...
+    ```
+
+!!! tip "Loading Multi Active Satellites correctly"
+    dbtvault provides custom materialisations, designed to load structures which contain deltas (such as multi active satellites, among other structures) 
+    in the correct way:
+    
+    - [vault_insert_by_period](../macros.md#vault_insert_by_period)
+    - [vault_insert_by_rank](../macros.md#vault_insert_by_rank)
+
+[Read more about incremental models](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/configuring-incremental-models/)
+
+### Adding the metadata
+
+Let's look at the metadata we need to provide to the [ma_sat](../macros.md#ma_sat) macro.
+
+#### Source model
+
+The first piece of metadata we need is the source model. This step is easy, as in this example we created the 
+staging layer ourselves.  All we need to do is provide the name of stage table as a string in our metadata 
+as follows.
+
+=== "dbt_project.yml"
+
+    ```yaml
+    ma_sat_customer_details:
+      vars:
+        source_model: 'stg_customer_hashed'
+    ```
+
+!!! tip
+    See our [metadata reference](../metadata.md#multi-active-satellites-mas) for more ways to provide metadata
+
+#### Source columns
+
+Next, we define the columns which we would like to bring from the source.
+Using our knowledge of what columns we need in our ```ma_sat_customer_details``` table, we can identify columns in our
+staging layer which map to them:
+
+1. The primary key of the parent hub or link table,  which is a hashed natural key. 
+The `CUSTOMER_PK` we created earlier in the [staging](tut_staging.md) section will be used for `sat_customer_details`.
+
+2. The child dependent key, `CUSTOMER_PHONE`, that is part of the payload inside the raw [staging](../macros.md#stage) layer.  
+3. A hashdiff. We created `HASHDIFF` in [staging](tut_staging.md) earlier, which we will use here.
+4. Some payload columns: `CUSTOMER_NAME`, `CUSTOMER_PHONE` which should be present in the 
+raw staging layer via an [stage](../macros.md#stage) macro call.
+5. An `EFFECTIVE_FROM` column, also added in staging. 
+6. A load date timestamp, which is present in the staging layer as `LOAD_DATE`. 
+7. A `SOURCE` column.
+
+We can now add this metadata to the `dbt_project`:
+
+=== "dbt_project.yml"
```

Comment:
We need to update this section in ALL of the tutorial steps to not to include the dbt_project.yml example anymore. Might be best to link to the Metadata reference, with a short explanation in the tutorial step.

Reply:
This again seems like a task for a future bug fix PR including the required changes for all tutorials?

Reply:
Yes agreed